test/tools/IsolTestOptions.cpp

```diff
@@ -76,6 +76,8 @@ bool IsolTestOptions::parse(int _argc, char const* const* _argv)
 		return false;
 	}
 	enforceViaYul = true;
+	enforceGasTest = true;
+	enforceGasTestMinValue = 100000;
```

Comment:
Maybe we could have some global constants for these defaults? They get repeated a lot.

Reply:
True that, plus maybe will remove from few places as well

Reply:
After removing it is present only in one place, I think it is fine now